commcare_connect/opportunity/tables.py

```diff
@@ -254,6 +251,13 @@ class Meta:
             "incomplete",
         )
 
+    def render_details(self, record):
```

Comment:
why the rendering change? is it the same end result, because the `LinkColumn` is nice in that its defined with all the other columns rather than in a separate place

Reply:
The rendering has changed because earlier we were getting the `org_slug` from the record itself using `utils.A()`. Now, we're retrieving the org_slug from class-level properties. I think I tried using `utils.A()` to access class-level properties, but it didn't work.


FYI - This is primarily intended for Program Managers. It is not mandatory for them to be members of the Network Manager's organization, but they can still view all details related to opportunities created by the Program Manager.

Reply:
So 
```
args=[utils.A(self.org_slug), utils.A("opportunity__id"), utils.A("pk")],
```
doesn't work? Is that because you can't reference self there?

Reply:
yes